Cargo.toml

```diff
@@ -1,51 +1,80 @@
-[package]
-name = "blue-build"
+[workspace]
+members = [ "utils", "recipe","template"]
+
+[workspace.package]
 version = "0.8.0"
-edition = "2021"
 description = "A CLI tool built for creating Containerfile templates based on the Ublue Community Project"
+edition = "2021"
 repository = "https://github.com/blue-build/cli"
 license = "Apache-2.0"
 categories = ["command-line-utilities"]
+
+[workspace.dependencies]
+anyhow = "1"
+format_serde_error = "0.3.0"
+log = "0.4"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+serde_yaml = "0.9.30"
+typed-builder = "0.18.1"
+uuid = { version = "1.7.0", features = ["v4"] }
+
+[workspace.lints.rust]
+unsafe_code = "forbid"
+
+[workspace.lints.clippy]
+correctness = "warn"
+suspicious = "warn"
+perf = "warn"
+style = "warn"
+nursery = "warn"
+
+[package]
+name = "blue-build"
 build = "build.rs"
 
+version.workspace = true
+edition.workspace = true
+description.workspace = true
+repository.workspace = true
+license.workspace = true
+
 [dependencies]
-anyhow = "1"
-askama = { version = "0.12", features = ["serde-json", "serde-yaml"] }
-chrono = "0.4"
+blue-build_utils = { path = "./utils" }
+blue-build_recipe = { path = "./recipe" }
```

Comment:
Should these blue_build_*

The mismatch casing is odd

Reply:
We should keep it consistent with the root crate so how about using `blue-build-*` instead.

Reply:
Per conversation in discord, we're going with `-`.